Cargo.toml

```diff
@@ -89,6 +89,7 @@ git = "https://github.com/pingcap/murmur3.git"
 
 [dependencies.rocksdb]
 git = "https://github.com/pingcap/rust-rocksdb.git"
+rev = "9a1c83c5382fbaee8a5102213c711bbe52d71470"
```

Comment:
Can we remove this?

Reply:
I think this is a good idiom to keep. It is not good for us to modify Cargo.lock previously. Instead we should bump this version and let the compiler update the lock file.

Reply:
I don't think it's a good idea. The whole point of keeping `Cargo.lock` is to let cargo keep track of the commit hash, so one doesn't have to provide one manually. In the old way, simple command `cargo update -p rocksdb` will do all the dirty works for us.